lib/pure/includes/osenv.nim

```diff
@@ -90,7 +90,8 @@ else:
     else:
       return c_getenv(key) != nil
 
-  proc putEnv*(key, val: string) {.tags: [WriteEnvEffect].} =
+  proc putEnv*(key, val: string, overwrite: bool = true) {.tags: [
```

Comment:
let's defer this API addition (adding `overwrite` param) change to another PR; you can mention this as "future work" in your PR msg

Reply:
Sorry, does "PR msg" mean the description at the top?

Reply:
yes, description on top of https://github.com/nim-lang/Nim/pull/18535#issue-693016862 (eg as i do here: https://github.com/nim-lang/Nim/pull/18244)